templates/_base/cookie-policy.html

```diff
@@ -1,6 +1,5 @@
-<div class="cookie-policy">
-	<div class="wrapper">
-		<a href="?cp=close" class="link-cta">Close</a>
-		<p>We use cookies to improve your experience of ubuntu.com. By continuing to explore without changing your settings, you are agreeing to accept them. To learn how to change these settings, please see our <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy#cookies" class="external">privacy policy</a>.</p>
-	</div>
+<div class="p-notification">
+  <p class="p-notification__response">
+    We use cookies to improve your experience of ubuntu.com. By continuing to explore without changing your settings, you are agreeing to accept them. To learn how to change these settings, please see our <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy#cookies" class="p-link--external">privacy policy</a><a href="#" class="p-notification__action">Dismiss</a>.
```

Comment:
This is using two versions of cookie policy and global nav. We should use the new version if possible.